api/handlers_test.go

```diff
@@ -132,7 +132,7 @@ func TestUserLimitsGetFromTier(t *testing.T) {
 			}
 		}()
 		// The call that we expect to log a critical.
-		_ = userLimitsGetFromTier("", math.MaxInt64, false, true)
+		_ = userLimitsGetFromTier("", math.MaxInt, false, true)
```

Comment:
The target field is `int`, so we should use `math.MaxInt`. Even if we only ever run this on 64 bit machines, we should still stick to the correct types.

Reply:
This changed because it was breaking the lint. Maybe my local go version is behind because I didn't see this as a valid option.

Reply:
@MSevey can you show me the error you're gerring? It seems that we have some differences in our setups which shouldn't be possible. I am really interested in chasing this one because having consistently identical working environments saves a ton of time and effort in the long run. I triple-checked that `make lint` doesn't show me any warnings or errors.